src/VirtualClient/VirtualClient.Actions/HPLinpack/HPLinpackExecutor.cs

```diff
@@ -84,7 +77,7 @@ public string ProblemSizeN
         {
             get
             {
-                return this.Parameters.GetValue<string>(nameof(HPLinpackExecutor.ProblemSizeN), Environment.ProcessorCount * 10000);
+                return this.Parameters.GetValue<string>(nameof(this.ProblemSizeN), Environment.ProcessorCount * 1000);
```

Comment:
Is this a breaking change with previous releases? Meaning if user switch version, the number they are getting is different

Reply:
Nope even for the previous release, the default value of ProcessorCount * 10000 for large processor count is exceeding memory and we are facing memory issues.

Reply:
Does that mean linpack is not linear scaling? otherwise why for smaller processor it's working but not for larger ones? Azure has linear memory/cpu